Unpopulated Areas 2010 Approx (#147)
agrc/metadata-asset-tracking-tool: metadata/demographic/unpopulated_areas_2010_approx.md

# Title

Utah Unpopulated Areas 2010 Approx

## ID

87112b87-1e64-49e7-a291-e35e188ca250

## Brief Summary

Polygon dataset of unpopulated areas in Utah dissolved as a single feature.

## Summary

> This dataset contains a polygon representing unpopulated areas in Utah as of the 2010 Census. For 2020 Census data products, please see the [Demographic Data Index](https://gis.utah.gov/products/sgid/demographic/) available from the SGID.

Reviewer: ```suggestion
This dataset contains a polygon representing unpopulated areas in Utah, derived from 2010 Census and addressing data. For 2020 Census data products, please see the [Demographic Data Index](https://gis.utah.gov/products/sgid/demographic/) available from the SGID.
```
Clarifying to note that this is a derivative product/result of our own analysis.

## Description

### What is the dataset?

> Census blocks are small geographic zones through which population data can be collected and organized. These blocks are often bounded by features such as roads, rivers, or property lines. This dataset includes Census blocks in unpopulated areas of Utah as of the 2010 Census that have been dissolved into a single polygon.

Reviewer: Let's de-emphasize the census block mentions due to uncertainty of data creation.  I think the PopBlockAreas2010_Approx was probably used to help create this, but we should focus more in this data and less on the blocks.
```suggestion
This dataset is a representation of Utah areas that have little to no population, buildings, or addresses, dissolved into a single polygon.
```

### What is the purpose of the dataset?

> Geography is a fundamental aspect of the Census, providing the framework for the once-a-decade count of population and housing. You can read more about some of the key geography changes in the Kem C. Gardner Policy Institute's 2020 Census Geography [Blog](https://gardner.utah.edu/blog/blog-whats-new-in-utahs-census-2020-geography/) and [fact sheet](https://d36oiwf74r1rap.cloudfront.net/wp-content/uploads/Geog-FS-Mar2021.pdf).

Reviewer: ```suggestion
This dataset was created to represent the portion of Utah with little to no permanent population.  Geography is a fundamental aspect of the Census, providing the framework for the once-a-decade count of population and housing. You can read more about some of the key geography changes in the Kem C. Gardner Policy Institute's 2020 Census Geography [Blog](https://gardner.utah.edu/blog/blog-whats-new-in-utahs-census-2020-geography/) and [fact sheet](https://d36oiwf74r1rap.cloudfront.net/wp-content/uploads/Geog-FS-Mar2021.pdf).
```

### What does the dataset represent?

> This dataset displays areas that contain little or no residential districts or developed areas as a single polygon feature. This dataset does not contain demographic data.

Reviewer: ```suggestion
This dataset displays the portion of Utah that contains little or no residential or developed areas, represented as a single polygon feature. This dataset does not contain demographic data.
```
Rewording

### How was the dataset created?

> UGRC created this dataset using the original 2010 Census Blocks from the US Census Bureau. We used aerial imagery to determine unpopulated areas of Utah. This dataset was updated in 2015 using new aerial photography and address point locations.

Reviewer: ```suggestion
UGRC created this dataset using parts of the original 2010 Census data from the US Census Bureau. Populated census blocks and a buffer around known addresses were used erase populated areas within the state, thus leaving behind the unpopulated areas in this dataset.
```
More rewording due to uncertainty in the data creation.

### How reliable and accurate is the dataset?

> Unpopulated areas shown in this dataset are approximate only. This layer is for historical reference and does not reflect current population numbers or Census block boundaries. Please reach out to [our team](https://gis.utah.gov/contact/) with questions or concerns about this dataset.

Reviewer: ```suggestion
Unpopulated areas shown in this dataset are approximate only. This layer is for historical reference and does not reflect current population numbers or Census data. Please reach out to [our team](https://gis.utah.gov/contact/) with questions or concerns about this dataset.
```
Removed reference to census blocks, due to some uncertainty in how it was created.
